brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post layout - Content padding 0 issue, Line height in editor and starting offset option.

### Description

- Link - https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/120ab759-bb2c-40b2-a5f4-942908090d72?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Offset Starting Post, should add this option ( Done )
- For Content Padding - 
   - Content padding setting is not saving when 0 is the value( Done - https://share.getcloudapp.com/4guKw2qw )
   - Create a Post block in Master and switch to our branch to check value set in master should replicate in the editor when you switch.
   - Update the value and check it is working on the front end and refresh the editor it should replicate the updated value in the content padding option.
   - Set value for responsive devices and check
   - No issue in debug log after frontend refresh
- Title font line-height does not look right in the backend. ( Done - https://share.getcloudapp.com/E0ug5rej )

### Checklist:
